### ISSUE TITLE
The theme.css file inside assets in extensions folder is not applying

### ISSUE THREAD

**Issue by Prashant-202** [iskurbanov/theme-app-extension-react#36]
# Issue summary

- `@shopify/shopify-app-express` version:
- Node version: 20.0.0
- Operating system: Ubuntu

## Expected behavior

The rules inside the external CSS will be loaded and applied to the elements.

## Actual behavior

The CSS file is loaded and showing inside the network but the rules are not applied to it.

## Steps to reproduce the problem

1. Write some CSS code in the theme.css file inside the frontend/entrypoints folder.
2. Give classname to the element in which we want to apply the css rules.
3. Running the "npm run deploy" command and deploying the app
![SS_02](https://github.com/iskurbanov/theme-app-extension-react/assets/139455758/168ba1de-67b4-4bc1-83e3-0a4706e3e8b0)
![SS_03](https://github.com/iskurbanov/theme-app-extension-react/assets/139455758/9f448ce4-2eef-4cb7-94ca-cb25af366909)
![SS_01](https://github.com/iskurbanov/theme-app-extension-react/assets/139455758/02dec0aa-d601-44b6-801e-67db9e208967)

**Comment by wrightmk** [comment]
bump

**Comment by iskurbanov** [comment]
Yes, there must have been a small change in the build process that doesn’t acknowledge the tailwind styles when referenced separately. 

You need to use the cssInjectedByJsPlugin to make it work. Npm install the package and then add it like this:

![image](https://github.com/iskurbanov/theme-app-extension-react/assets/12405055/6c31b740-e7d6-48f0-8681-c9854b0c12be)

**Comment by wrightmk** [comment]
This works! thanks!

**Comment by 921kiyo** [comment]
Thank you! This fixed the issue for me as well!

**Comment by bakikucukcakiroglu** [comment]
Thank you! That solved my problem as well. However you need to indicate it somewhere because I spent more then 5 hours to fix it.

**Comment by har1zarD** [comment]
I have a problem with Tailwind. When I add 10 different app blocks, Tailwind does not apply to any of them. But if I add one more app block that is identical to one of those 10, Tailwind immediately applies to all app blocks. So I need to add 2 identical app blocks to get Tailwind classes. @iskurbanov
